packages/examples/examples/typescript/webpack.config.ts

```diff
@@ -0,0 +1,50 @@
+import SnapsWebpackPlugin from '@metamask/snaps-webpack-plugin';
```

Comment:
How much faster is building the examples with Webpack?

I'm wondering if it gets too complicated compared to using the CLI.